modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -137,6 +137,7 @@ conf = {{{packageName}}}.Configuration(
     """
 
     _default = None
+    __debug = False
```

Comment:
There's no reason to make it a class attribute, can you move this as an instance attribute line 217?

Reply:
done